pykern/pkcli/test.py

```diff
@@ -188,8 +194,14 @@ def _try(output, restartable):
                     env=_env(restartable),
                 )
                 o = pkio.read_text(output)
-                if _COROUTINE_NEVER_AWAITED.search(o):
-                    pkio.write_text(output, o + _FAILED_ON_WARNINGS)
+                if m := _COROUTINE_NEVER_AWAITED.search(o):
```

Comment:
There may be multiple of these so you'll probably want to include that in the error message, e.g. 

```txt
Failed due to "coroutine was never awaited" here:
<text>
```